terraform/accounts/main/main.tf

```diff
@@ -49,7 +49,10 @@ module "sops_credentials" {
 }
 
 module "jenkins" {
-  source              = "../../modules/jenkins"
-  aws_main_account_id = "${var.aws_main_account_id}"
-  aws_sub_account_ids = "${var.aws_sub_account_ids}"
+  source                     = "../../modules/jenkins"
+  aws_main_account_id        = "${var.aws_main_account_id}"
+  aws_sub_account_ids        = "${var.aws_sub_account_ids}"
+  jenkins_security_group_ids = "${var.jenkins_security_group_ids}"
+  jenkins_2_key_name         = "${var.jenkins_2_key_name}"
```

Comment:
It might be worth getting that jenkins 2 branch of the credentials repo merged too? Should make things a bit smoother.

Reply:
Yeah trouble is I'm still having to hack on that fairly constantly. I think I probably need to move a bunch of things into a separate file so that it's easier to manage. Not sure...

Reply:
oh, you're telling me this can't land without the creds repo landing first? argh :)

Reply:
Yeah it could land, it just means that we'd have stuff on the master branch that wouldn't succesfully plan or apply using the master credentials branch. It feels like we should be keeping things in sync.